src/OmniSharp.Host/WorkspaceInitializer.cs

```diff
@@ -29,6 +29,8 @@ public static void Initialize(IServiceProvider serviceProvider, CompositionHost
             projectEventForwarder.Initialize();
             var projectSystems = compositionHost.GetExports<IProjectSystem>();
 
+            workspace.EditorConfigEnabled = options.CurrentValue.FormattingOptions.EnableEditorConfigSupport;
```

Comment:
it would be good to also reset this flag here https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp.Host/WorkspaceInitializer.cs#L62-L65
this way it should be effective in real time when options change (since they are watched)

Reply:
This would be useful if there were a way to force all the projects to update. Is this option worth all this extra complexity? I would assume the presence of an .editorconfig would be the indicator of whether a user wanted .editorconfig support.

Reply:
you are right we can solve this systematically separately as there are other areas that would benefit from that